src/components/AFEpage/styles.js

```diff
@@ -0,0 +1,37 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles((theme) => ({
+  Grid_Space: {
+    margin: "30px 0px 30px 0px",
```

Comment:
Since top/bottom are the same and so is the right/left, you could use 2 numbers here:
```suggestion
    margin: "30px 0px",
```